python-spec/pyproject.toml

```diff
@@ -9,6 +9,8 @@ dynamic = ["version"]
 readme = "./README.md"
```

Comment:
Once this package is available, it will likely get confused with `tiledbsoma`.  I am wondering if we need the README (or something) to point people toward that repo/package?

Reply:
A good idea for a followup.